nodewatcher/modules/frontend/api/templates/rest_framework/base.html

```diff
@@ -1,21 +1,230 @@
-{% extends "rest_framework/base.html" %}
```

Comment:
So then we are now simply overriding their template fully? Hmm. Should we then extend from our base template?